src/base/bittorrent/session.cpp

```diff
@@ -269,6 +288,8 @@ Session::Session(QObject *parent)
     , m_maxUploadsPerTorrent(BITTORRENT_SESSION_KEY("MaxUploadsPerTorrent"), -1, lowerLimited(0, -1))
     , m_isUTPEnabled(BITTORRENT_SESSION_KEY("uTPEnabled"), true)
     , m_isUTPRateLimited(BITTORRENT_SESSION_KEY("uTPRateLimited"), true)
+    , m_utpMixedMode(BITTORRENT_SESSION_KEY("uTPMixedMode"), m_isUTPEnabled, clampValue(0, 1))
```

Comment:
>Don't know what to say now... I would still use a textual representation.

OK, that is not the main point now, I just wanted to say it's still debatable IMO.

>Apparently, @evsh used something different... maybe, just enums (not enum classes). I think, we can use his code as base for cases of enumerable settings.

I tried both and it doesn't matter much which I use, combination of `enum, enum class, Q_ENUM_NS, Q_ENUM`.

I looked it a bit more closely, at
https://github.com/qbittorrent/qBittorrent/blob/master/src/base/settingvalue.h#L99
so it uses `QMetaEnum::fromType()` "magic" and Qt doc states 
>The enum needs to be declared with Q_ENUM.

However, in this case the enum is in namespace scope, so should use `Q_ENUM_NS` instead, but the requirement `Qt >= 5.8` doesn't suit us now as I said before.

... unless I deploy some workaround like this:
```c++
struct Wrapper{
  enum class ChokingAlgorithm : int
  {
    FixedSlotsChoker = 0,
    RateBasedChoker = 1
  };
  Q_ENUM(ChokingAlgorithm)
};
using ChokingAlgorithm = Wrapper::ChokingAlgorithm;
```
This is the best I can do now...